collection_requirements.rst

```diff
@@ -217,7 +217,30 @@ We should avoid FQCN / repository names:
 Licensing
 =========
 
-At the moment, ``module_utils`` must be licensed under the `BSD-2-clause <https://opensource.org/licenses/BSD-2-Clause>`_ or `GPL-3.0-or-later <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_ license and all other content must be licensed under the `GPL-3.0-or-later <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_.  We will have a list of other open source licenses which are allowed as soon as we get Red Hat's legal team to approve such a list for us.
+**Note**: The guidelines below are more restrictive than strictly necessary.  We will try to add
+a larger list of acceptable licenses once we have approval from Red Hat Legal.
+
+There are three types of plugin code in collections which licensing has to address in different
+ways:
+
+:modules: must be licensed compatibly with the `GPL-3.0-or-later <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_
+:module_utils: must be licensed compatibly with the `GPL-3.0-or-later
+               <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_.  Ansible itself typically
```

Comment:
I there something missing in this sentence? I would expect something like "Ansible itself typically **uses** ...", but I am not a native speaker.

Reply:
Yes, thanks for catching that!